src/include/duckdb/execution/operator/order/physical_order.hpp

```diff
@@ -33,6 +33,10 @@ class PhysicalOrder : public PhysicalOperator {
 	void GetData(ExecutionContext &context, DataChunk &chunk, GlobalSourceState &gstate,
 	             LocalSourceState &lstate) const override;
 
+	bool IsOrderPreserving() const override {
+		return false;
```

Comment:
Is this correct? Won't this lead to incorrect parallel result set materialization?

Reply:
This is correct. It sounds confusing, but the `PhysicalOrder` does not preserve the insertion order - the data is completely reordered.